src/task_list.dart

```diff
@@ -0,0 +1,183 @@
+
+import'dart:io';
+import 'task.dart';
+
+
+List <Task> All_tasks = [] ;
+
+
+void add(){
+
+  late Task new_task =Task.create();
+ 
+  new_task.ID = All_tasks.length;
+
+  stdout.write('task title: ');
+  new_task.Title = stdin.readLineSync()!;
```

Comment:
Should add null safety to this
```suggestion
    String? title = stdin.readLineSync() ?? ""
    if(title == "")
    {
        stdout.write("Title Invalid")
    }
```

Maybe add a while loop or a try catch to keep the user in this state until a title is given